app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -7,12 +7,36 @@
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
+function require_ev_all() {
+    for rev in $@ ; do
+        if [[ -z ${!rev} ]]; then
+            echo ${rev} is not set
+            exit 1
+        fi
+    done
+}
+
+function require_ev_one() {
+    for rev in $@ ; do
+        if [[ ! -z ${!rev} ]]; then
+            return
+        fi
+    done
+    echo One of $@ must be set
     exit 1
-fi
+}
+
+KUBELET_VERSION="${KUBELET_VERSION:-${KUBELET_TAG}}"
+KUBELET_IMG="${KUBELET_ACI}"
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+require_ev_one KUBELET_IMG KUBELET_TAG
```

Comment:
Yes, please don't change the variable names. For consistency sake etcd should be updated to match kublet since kublet was first, not the other way around. @philips requested that change at a recent os meeting.